packages/address-book-controller/src/AddressBookController.ts

```diff
@@ -121,14 +98,19 @@ export type AddressBookControllerContactDeletedEvent = {
   payload: [AddressBookEntry];
 };
 
+const MESSENGER_EXPOSED_METHODS = ['list', 'set', 'delete'] as const;
+
+type AddressBookControllerMethodActions = MessengerMethodActions<
```

Comment:
Since we are generating all of the types individually, do you think we need these utility types anymore? Maybe when the script generates the `*-method-action-types.ts` file it can assemble and export a `*MethodActions` type which is a union of the individual types it's generated, and then we could import that union in this file? Then we could remove `MessengerMethodActions`, `MessengerMethodAction` and `ExtractActions` from `Messenger.ts`.